lib/ui/pages/news_list/news_list_page.dart

```diff
@@ -11,21 +11,31 @@ class _NewsPageState extends State<NewsListPage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Column(
-        children: [
-          Padding(
-            padding: EdgeInsets.all(26.0),
-            child: NewsListItem(
-              news: NewsViewEntity(
-                  title: "Egypt Is on Edge as Security Tightens Over Protests",
-                  date: "Sept. 26, 2019",
-                  time: "1:41 PM",
-                  imageUrl: "https://i.picsum.photos/id/9/250/250.jpg"),
-            ),
+      body: SafeArea(
+        child: Padding(
+          padding: EdgeInsets.all(26.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                "Middle East",
```

Comment:
wszystkie stringsy przydałoby się wyrzucić do osobnej klasy, albo jako jakiś InheritedWidget (coś w stylu Theme.of tylko, że nasze), albo po prostu jako consty

Reply:
chyba, że od razu zajmujemy się ewentualnym tłumaczeniem wszystkich tesktów to można do tego użyć klasy Localizations, dać ją do jakiegoś stanu bazowego razem ze stylami, kolorami, nawigacją etc, który byłby dziedziczony po wszystkich widokach

Reply:
![image](https://user-images.githubusercontent.com/24854180/80516839-674cdb80-8984-11ea-9e11-e184f0b5d3a5.png)

Reply:
Póki co te wszystkie teksty pełnią funkcję mocków, więc imo jakieś tam wydzielanie i tłumaczenie ich to mocny overengineering imo :P Co do mechanizmu to proponowałbym w ramach osobnego taska ogarnięcie jak te stringsy byśmy trzymali w apsie z uwzględnieniem mechanizmu tłumaczeń, co ty na to? ;)

Reply:
zgadzam sie, że na teraz nie ma sensu robienie tych mechanizmów. Ale tak, task na wydzielenie stringów do osobnego pliku by się przydał